roles/uperf/templates/workload.yml.j2

```diff
@@ -84,17 +84,19 @@ spec:
         command: ["/bin/sh", "-c"]
         args:
 {% if workload_args.serviceip is sameas true %}
-          - "export serviceip=true;
```

Comment:
Why did this change for ServiceIP?

Reply:
^ this way it will only export serviceip=true. have added it outside the if condition below so that it catches the serviceip=false too

Reply:
ack I would prefer we break this out of the current PR and put it into a different PR...

Reply:
done